frontend/src/app/app.component.ts

```diff
@@ -1,16 +1,45 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { NavigationCancel, NavigationEnd, NavigationError, NavigationStart, Router } from '@angular/router';
+import { SocialAuthService, SocialUser } from '@abacritt/angularx-social-login';
 import { SpinnerService } from '@core/services/spinner.service';
 
 @Component({
     selector: 'app-root',
-    template: '<app-loading-spinner></app-loading-spinner><router-outlet></router-outlet>',
+    templateUrl: './app.component.html',
+    styleUrls: ['./app.component.sass'],
 })
-export class AppComponent {
-    constructor(private router: Router, private spinner: SpinnerService) {
+export class AppComponent implements OnInit {
+    loginForm!: FormGroup;
+
+    socialUser!: SocialUser;
+
+    isLoggedIn?: boolean;
```

Comment:
maybe it is better to have explicit access modifiers (private, public...) as we do in C#

Reply:
updated